training/checkpoint.py

```diff
@@ -36,7 +46,42 @@ def load_checkpoint(checkpoint_path, model, optimizer, train_loader):
     return model, optimizer, iteration, epoch
 
 
-def warm_start_model(checkpoint_path, model, ignore_layers=["embedding.weight"]):
+def transfer_symbols_embedding(original_embedding_weight: torch.Tensor, embedding_layer, original_symbols: Optional[list], new_symbols: list):
```

Comment:
Please add a docstring for this function

Reply:
little busy, do it for me if you can, overwise wait for me

Reply:
I'll give it a go, we'll also need unit tests

Reply:
Suggestion: 

"""
Transfer embedding information from transfer learning model to reduce embedding time.
If symbol is not found it is initialised with mean/std.

Parameters
----------
original_embedding_weight : Torch.tensor
    Checkpoint embeddings
embedding_layer : ???
    Model embedding layer
original_symbols : list (optional)
    list of current symbols (defaults to NVIDIA_ALPHABET)
new_symbols : list
    list of text symbols used by the fresh model currently loaded
"""